mod_deps.properties

```diff
@@ -1,8 +1,9 @@
 # mod_deps should be 'after:<mod identifier>' (unquoted, semicolon-separated, \escaped for newline, see example below)
-bopver=1.10.2-5.0.0.2067
-acver=1.10.2-1.9.2.5
-depstring=after:BiomesOPlenty@[5.0.0.2067,);\
-            after:AbyssalCraftAPI@[1.8,);\
+bopver=1.10.2-5.0.0.2085
+acver=1.10.2-1.9.2.9
+depstring=after:BiomesOPlenty@[5.0.0.2085,);\
+            after:AbyssalCraftAPI@[1.8.1,);\
             after:BiomesYouGo@[3.0.2,);\
-            after:flowercraft;\
-            after:sugiforest
+            after:flowercraft@[6.01,);\
```

Comment:
There was a reason I didn't add a version string to Flowercraft and Sugiforest, although I can't remember what it was! 😜 I'll need to refresh my memory, but I think it had something to do with dodgy version numbers in the JAR not matching up or something.